### PR TITLE
Standard Lint and clean up a few things

### PR DESCRIPTION
**Pull request by hiimbex**
`whiteList` -> `filesNotToLint` because 1. read an article about the racial implications of whitelist/blacklist and 2. `filesNotToLint` is a clearer name anyways. Also standard lints.

After this gets merged, any reason we can't deploy a version of this and start using it on probot/probot? And list it on probot.github.io ? I think even just as standard linting it has a lot of potential.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +17,8 @@
 ```yml
 # Configuration for linter - https://github.com/probot/linter
 
-# a whitList of files the linter will NOT be applied to
-whiteList:
+# a list of files the linter will NOT be applied to
+filesNotToLint:
```
Comment (bkeepers) [inline README.md]: Good call, and thanks for pointing out the horribleness of this word. I never thought about it, but will try to never use it again.

What about `exclude`?

File: package.json [hunk 2]
```diff
@@ -1,31 +1,30 @@
 {
   "name": "probot-linter",
   "version": "1.0.0",
-  "description": "{{{ description }}}",
-  "author": "{{{ author }}}",
+  "description": "automatically lints code",
+  "author": "Bex Warner and Brandon Keepers",
```
Comment (bkeepers) [inline package.json]: I think author is intended to be one person (which is weird): https://docs.npmjs.com/files/package.json#people-fields-author-contributors

You've done more on this plugin than I have at this point, so feel free to just set your name there. 😁

**Review by bkeepers — APPROVED** [review]
Looks good. A couple comments, but feel free to merge whenever you're happy with it!

File: package.json (hunk 2 above)
Comment (hiimbex) [reply package.json]: Super weird. What about using the contributors field instead?

```
"contributors": [
    {
      "name": "name"
    },
    {
      "name": "name"
    }
],
```
There are more fields that can be included in there, but they don't really matter.

Comment (bkeepers) [reply package.json]: 👍 works for me

**Comment by bkeepers** [comment]
@hiimbex gentle nudge here. Do you plan to get back to this, or would you prefer someone else take it over?

**Comment by hiimbex** [comment]
@bkeepers gentle wave here! Been very busy lately but still trying to stay involved with probot things where I can! :)

**Pull request merged**